src/ado/reprun_dataline.ado

```diff
@@ -33,28 +33,20 @@ cap program drop   reprun_dataline
       *Get the states to be checked
       local srng "`c(sortrngstate)'"
       datasignature
-      local dsig "`r(datasignature)'"
+      local dsum "`r(datasignature)'"
       * Trim looptracker to remove excessive spaces
       local loopt = trim("`looptracker'")
-      
-      * Handle data line
-      local output = substr(`"`datatmp'"',1,strrpos(`"`datatmp'"',".txt"))
-      local data = "`lnum'`looptracker'"
-      local data = subinstr("`data'"," ","_",.)
-      local data = subinstr("`data'",":","-",.)
-      if `run' == 1 {
-        cap mkdir "`output'"
-        save "`output'/`data'.dta" , replace emptyok
-        local srngcheck = _rc
-      }
-      if `run' == 2 {
-        local output = subinstr(`"`output'"',"run2","run1",.)
-        cap cf _all using "`output'/`data'.dta"
-        local srngcheck = _rc
-      }
+
+      * Handle data state
+      local output = subinstr(`"`datatmp'"',".txt",".csv",.)
+        cap export delimited using "`output'.dta" , replace
+        cap qui checksum "`output'.dta"
```

Comment:
```suggestion
        cap export delimited using "`output'" , replace
        cap qui checksum "`output'"
```

99% sure this was just left from previous version, but editing in suggestion to double check with you

Reply:
Honestly this could be a tempfile